pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -92,10 +97,10 @@ func (o *capability) HealthReport() map[string]error {
 }
 
 type workflowConfig struct {
-	AggregationMethod string      `mapstructure:"aggregation_method"`
```

Comment:
Are we losing portability by using irregular struct tags like `jsonschema`?

Does the experimental v2 API meet our needs?
- https://pkg.go.dev/github.com/go-json-experiment/json
- https://github.com/golang/go/discussions/63397

Reply:
The `jsonschema` struct tag doesn't affect json (de)serialization, it instructs the jsonschema package on how to generate a JSON schema. Unfortunately that new API doesn't help us here

Reply:
Are the schemas a standard format, or specific to `jsonschema`?

Reply:
They are a standard https://json-schema.org/